### ISSUE TITLE
FireTV Stick

### ISSUE THREAD

**Issue by DreamStatic** [mdhiggins/PlexAutoSkip#27]
Hello, first let me just say, this was awesome of you to create and share!!! Thank you so much!

My server runs on a Windows 10 machine and I am also running PlexAutoSkip script on the same machine. 

I have a ShieldTV in my living room and PlexAutoSkip works wonderfully on it. 

However, in our bedroom, we have a FireTV 4k Stick and it does not skip intros on it. 
The log appears to see the player, as it lists the episode and that it found a skippable marker intro. 
And shows an INFO line stating it is sending the Seeking Plex for Android ........ 475698 to 486138. 
And then some Traceback lines ....
then ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it.

Just curious if anyone has run into this and found a remedy? Thanks again for a cool share.

**Comment by mdhiggins** [comment]
https://github.com/mdhiggins/PlexAutoSkip/wiki/Troubleshooting

I would try the usual troubleshooting steps, ensure same subnet and same account etc, try restarting and relogging

A full set of debug level logs would also be helpful, could infer more information but the fact that its a connection refused error might mean you need to make sure the accounts are the same and get a fresh Plex token for your authentication for PlexAutoSkip

https://github.com/mdhiggins/PlexAutoSkip/wiki/Debug-Level-Logging

Overall though FireTV is supported and does work (though its definitely one of the less consistent plex companion client implementations) so I suspect this is something with your setup causing an issue. You can also take a look at this thread on some other issues that FireTV users had

EDIT:

https://github.com/mdhiggins/PlexAutoSkip/issues/3

Referenced the wrong issue originally, look at the last few posts on that issue

**Comment by DreamStatic** [comment]
Thanks for the tips. I was able to get it working. 

For anyone having this issue, double check on the FireTV the IP it shows. That is what made me realize the issue, as it was showing 192.168.2.168. The issue ended up being my mesh network. Somehow, my second child mesh router was on a different IP/DHCP range than the master mesh router. My server was connected to the master mesh router and the problem FireTV was connected to the child mesh router.

Master mesh router IP/DHCP was on range: 192.168.1.1
Child mesh router IP/DHCP was on range: 192.168.2.1

So PlexAutoSkip was actually pinging the IP of the child mesh router instead of the actual FireTV IP. In the end, I reset the child mesh router back to factory settings and removed/readded it back to the rest of the mesh network. Ensuring it stayed in the 192.168.1.1 IP/DHCP range. 

Once I did that, BOOM!!! PlexAutoSkip found the true FireTV session at 192.168.1.168 and intro skip started working like a dream. 

Thanks again for this awesome feature addition to Plex. LOVE IT!!
